ui/lib/core/addon/helpers/has-capability.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/**
+ * helper to get capabilities from map for given item by id
+ * useful in list views when capabilities are fetched for each item
+ * provide the full map of capabilities along with the id of the item and capabilities to be evaluated (read, update etc)
+ * multiple capabilities will return true if any of the capabilities are true by default
+ * to check if all provided capability types are true, set the all argument to true
+ * usage example: {{has-capability this.capabilities "delete" "update" id=item.id all=true}}
```

Comment:
Thanks for including an example! This is super helpful, would the actual use need to wrap the capability types in an array helper? 
```suggestion
 * usage example: {{has-capability this.capabilities (array "delete" "update") id=item.id all=true}}
```

Reply:
No actually the positional arguments in the template are passed to the helper as an array so the capabilities need to be the first arg and then the remaining args are spread to get the types.